src/Microsoft.TestPlatform.Common/Resources/xlf/Resources.cs.xlf

```diff
@@ -279,6 +279,11 @@
         <target state="translated">Nepovedlo se načíst rozšíření ze souboru {0}. Další informace získáte pomocí parametru /diag.</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="AccessDenied">
+        <source>{0}. Access denied while trying to create "TestResults" folder in mentioned location. You are getting this exception because you are running vstest.console.exe from a folder which requires having write access. To solve the issue : please  run vstest.console.exe from a folder where you have write privileges to that folder. For more information, please look at the error message :</source>
```

Comment:
It looks like those error messages for different cultures are not up to date. You may want to run the `build.cmd` script to regenerate them.

Reply:
Fixed.